test/integration/install_scripts_int_test.go

```diff
@@ -198,22 +202,28 @@ func (suite *InstallScriptsIntegrationTestSuite) TestLegacyInstallShInstallMulti
 	}
 	suite.OnlyRunForTags(tagsuite.InstallScripts, tagsuite.Critical)
 
-	ts := e2e.New(suite.T(), false)
+	ts := e2e.New(suite.T(), true)
```

Comment:
Was this for debugging?
```suggestion
	ts := e2e.New(suite.T(), false)
```